test/testConfigure.js

```diff
@@ -475,7 +492,17 @@ describe('Configure', function() {
     configure();
     return enterUsernamePassword()
     .then(function() {
-      return await('Deleting temporary GitHub token for getting Travis token…');
+      return await('Deleting temporary GitHub token for getting Travis token… done!');
+    })
+    .then(complete);
+  });
+
+  it('creates permanent GitHub token', function() {
+    nockBasicPostAuthFlow();
+    configure();
+    return enterUsernamePassword()
+    .then(function() {
+      return await('Creating permanent GitHub token for Travis to push to the repository… done!');
```

Comment:
Nit: maybe we could collapse some of these tests in a single one that checks all the output?